Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM centos:7
+FROM centos:7.3.1611
```

Comment:
maybe a comment here along the lines of "Update the README when this base image and/or the version of lvm2 (below) is updated"?